modules/platforms/dotnet/Apache.Ignite.Tests/Sql/SqlTests.cs

```diff
@@ -519,7 +519,7 @@ public async Task TestCustomDecimalScale()
             await using var resultSet = await Client.Sql.ExecuteAsync(null, "select cast((10 / ?) as decimal(20, 5))", 3m);
             IIgniteTuple res = await resultSet.SingleAsync();
 
-            Assert.AreEqual(3.33333m, res[0]);
+            Assert.AreEqual(3.3m, res[0]);
```

Comment:
it's should be either 3.00000 or 3.33333, but not 3.3

Reply:
fixed